src/main/java/fi/otavanopisto/kuntaapi/server/integrations/mwp/MwpIdProvider.java

```diff
@@ -1,5 +1,6 @@
 package fi.otavanopisto.kuntaapi.server.integrations.mwp;
 
+import javax.enterprise.context.Dependent;
 import javax.enterprise.context.RequestScoped;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.enterprise.context.RequestScoped'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)